src/main/java/me/redstom/privaterooms/commands/ConfigCommand.java

```diff
@@ -77,15 +81,15 @@ public CommandData command() {
           .addSubcommands(
             new SubcommandData("name", "Rename the channel")
               .addOption(OptionType.STRING, "name", "The new name of the channel", true),
```

Comment:
I don't really understand the point of renaming the `limit` to `user-limit` as the description of the subcommand already describes the command well.

Reply:
Le but était surtout de renommer la variable de maxUser à user-limit qui est le nom dans JDA, mais je vais revert le nom de la commande